content/en/datadog_cloudcraft/_index.md

```diff
@@ -19,60 +19,102 @@ Cloudcraft offers a powerful, live read-only visualization tool for cloud archit
+## Prerequisites
+
+{{< tabs >}}
+{{% tab "AWS" %}}
 
 - To access Cloudcraft in Datadog, you need the `cloudcraft_read` [permission](#permissions).
 - [Resource collection][2] must be enabled for your AWS accounts.
 - For the best experience, Datadog strongly recommends using the AWS-managed [`SecurityAudit`][5] policy, or the more permissive [`ReadOnlyAccess`][6] policy.
+
 - Viewing content on the [Security overlay][10] requires additional products to be enabled:
   - To view security misconfigurations and identity risks, [Cloud Security][3] must be enabled.
   - To view sensitive data, [Sensitive Data Scanner][12] must be enabled. For a user to turn the layer on, they must have the [`data_scanner_read`][13] permission.
 
 **Note**: Cloudcraft adapts to restrictive permissions by excluding inaccessible resources. For example, if you don't grant permission to list S3 buckets, the diagram excludes those buckets. If permissions block certain resources, an alert displays in the UI.
 
-<div class="alert alert-warning">Enabling resource collection can impact your AWS CloudWatch costs. To avoid these charges, disable <strong>Usage</strong> metrics in the <strong>Metric Collection</strong> tab of the <a href="https://app.datadoghq.com/integrations/amazon-web-services">Datadog AWS Integration</a>.<br/>
+<div class="alert alert-warning">
+Enabling resource collection can impact your AWS CloudWatch costs. To avoid these charges, disable <strong>Usage</strong> metrics in the <strong>Metric Collection</strong> tab of the <a href="https://app.datadoghq.com/integrations/amazon-web-services">Datadog AWS Integration</a>.
+</div>
+
+{{< img src="/infrastructure/resource_catalog/aws_usage_toggle.png" alt="The Usage service highlighted in the Metric Collection tab of the AWS integration tile" style="width:100%;" >}}
+
+[2]: /integrations/amazon_web_services/#resource-collection
+[3]: /security/cloud_security_management
+[5]: https://docs.aws.amazon.com/aws-managed-policy/latest/reference/SecurityAudit.html
+[6]: https://docs.aws.amazon.com/aws-managed-policy/latest/reference/ReadOnlyAccess.html
+[10]: /datadog_cloudcraft/overlays#security
+[12]: /security/sensitive_data_scanner
+[13]: /account_management/rbac/permissions/#compliance
+
+{{% /tab %}}
+{{% tab "Azure" %}}
+
+- To access Cloudcraft in Datadog, you need the `cloudcraft_read` [permission](#permissions).
+- You need the Datadog Admin Role, or any role with the `azure_configurations_manage` permission. See the [Azure setup][16] instructions for more information.  
 
-{{< img src="/infrastructure/resource_catalog/aws_usage_toggle.png" alt="AWS Usage toggle in account settings" style="width:100%;" >}}</div>
+- Enable [resource collection][14] for your Azure accounts:
+  1. Navigate to [**Integrations > Azure**][15].
+  2. Add your Azure subscription by selecting **+ Add New App Registration** if not already added.
+  3. Optionally, select the App Registration containing your Azure subscription.
```

Comment:
Could you confirm if step 3 is optional? It wasn't listed as optional before, and it looks like an app registration is always "selected" in the integration tile (when the integration is configured):

<img width="630" height="477" alt="azure-int-tile" src="https://github.com/user-attachments/assets/30972a5a-f1c0-4987-aeb0-8e82d33b6252" />

Reply:
Oh sorry I think this is confusing the way it's written - My understanding is there are two paths to Enabling Resource collection for Azure 
- One path is if you don't already have any subscriptions added to the integration, they need to click Add New App Registration
- The other path is if they already have subscriptions added to the integration, they would click the app registration they previously enabled then click the Resource collection tab. 

I can see how this is confusing! Let me rephrase this. Either way you are correct, it is indeed not an optional step :) Thanks for flagging this.